<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskClass</class>
 <widget class="QDialog" name="TaskClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	padding:3px;
}
TaskClass {
	background-color: rgb(240, 240, 240);
}</string>
  </property>
  <widget class="QTextEdit" name="descriptionEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>описание не заполнено</string>
   </property>
  </widget>
  <widget class="QComboBox" name="statusBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Сделать</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>В работе</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сделано</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Завершено</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="nameTaskLbl">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::TextFormat::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dealLbl">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QLabel" name="comLbl">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Связи</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="com">
      <property name="text">
       <string>Проект, сделка или контрашент</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>175</width>
     <height>484</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="inner">
      <item>
       <layout class="QVBoxLayout" name="dateLayout">
        <item>
         <layout class="QHBoxLayout" name="dateBeginLayout">
          <item>
           <widget class="QLabel" name="dateBeginLbl">
            <property name="text">
             <string>Дата начала</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dateBegin">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dateEndLayout">
          <item>
           <widget class="QLabel" name="deadLineLbl">
            <property name="text">
             <string>Крайний срок</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="deadLine">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="priorityLbl">
            <property name="text">
             <string>Приоритет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="priority">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QLabel" name="createLbl">
            <property name="text">
             <string>Создано</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="create">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_26">
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Счет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="creatorLbl">
            <property name="text">
             <string>Постановщик</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="creator">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="idLbl">
            <property name="text">
             <string>ID задачи</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="id">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <widget class="QLabel" name="completedDateLbl">
            <property name="text">
             <string>Завершено</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="completedDate">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_24">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Учитывать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="executorLbl">
            <property name="text">
             <string>Ответственный</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="executor">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_23">
          <item>
           <widget class="QLabel" name="editLbl">
            <property name="text">
             <string>Изменено</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="edit">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Оплачивается заказчиком</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_25">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Сумма</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="empLayout">
      <item>
       <widget class="QLabel" name="paidEmpLbl">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Оплачивается сотруднику</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_27">
        <item>
         <widget class="QLabel" name="considerEmpLbl">
          <property name="text">
           <string>Учитывать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="considerEmp">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_28">
        <item>
         <widget class="QLabel" name="amountEmpLbl">
          <property name="text">
           <string>Сумма</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="amountEmp">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
